pkg/kubelet/status/status_manager.go

```diff
@@ -786,17 +786,21 @@ func (m *manager) syncBatch(all bool) int {
 // syncPod syncs the given status with the API server. The caller must not hold the status lock.
 func (m *manager) syncPod(uid types.UID, status versionedPodStatus) {
 	// TODO: make me easier to express from client code
-	pod, err := m.kubeClient.CoreV1().Pods(status.podNamespace).Get(context.TODO(), status.podName, metav1.GetOptions{})
+	// TODO: it needs to be replaced by a proper context in the future
+	ctx := context.TODO()
+	logger := klog.FromContext(ctx)
+
+	pod, err := m.kubeClient.CoreV1().Pods(status.podNamespace).Get(ctx, status.podName, metav1.GetOptions{})
```

Comment:
Let's not make massive changes to other components. This PR should concentrate on migrating `kubelet/util` + as little as possible changes in the upper layer.